packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -229,44 +223,68 @@ const _generateStyles = (
 
     let newStyles: Record<string, any> = {};
     if (light) {
+        const focusHoverStyling = {
+            borderColor: error ? tokens.color.red : tokens.color.white,
+            borderWidth: tokens.spacing.xxxxSmall_2,
+            paddingLeft: adjustedPaddingLeft,
+            paddingRight: adjustedPaddingRight,
+        };
         newStyles = {
             default: {
                 background: error ? tokens.color.fadedRed8 : "transparent",
-                color: placeholder ? tokens.color.white50 : tokens.color.white,
+                color: error
+                    ? tokens.color.offBlack64
+                    : placeholder
+                    ? tokens.color.white50
+                    : tokens.color.white,
                 borderColor: error ? tokens.color.red : tokens.color.white50,
                 borderWidth: tokens.border.width.hairline,
-            },
-            focus: {
-                borderColor: error
-                    ? tokens.color.fadedRed8
-                    : tokens.color.white,
-                borderWidth: tokens.spacing.xxxxSmall_2,
-                paddingLeft: adjustedPaddingLeft,
-                paddingRight: adjustedPaddingRight,
-            },
-            active: {
-                paddingLeft: adjustedPaddingLeft,
-                paddingRight: adjustedPaddingRight,
-                borderColor: error ? tokens.color.red : tokens.color.fadedBlue,
-                borderWidth: tokens.border.width.thin,
-                color: placeholder
-                    ? mix(tokens.color.white32, tokens.color.blue)
-                    : tokens.color.fadedBlue,
-                backgroundColor: error
-                    ? tokens.color.fadedRed
-                    : tokens.color.activeBlue,
+                ":hover:not([aria-disabled=true])": focusHoverStyling,
+                ["@media not (hover: hover)"]: {
```

Comment:
**suggestion:** I think it would be good adding a comment here explaining why we add this custom media query.